src/plone/restapi/serializer/dxcontent.py

```diff
@@ -185,7 +188,15 @@ def __call__(self, version=None, include_items=True):
                 )(fullobjects=True)["items"]
             else:
                 result["items"] = [
-                    getMultiAdapter((brain, self.request), ISerializeToJsonSummary)()
+                    getMultiAdapter((brain, self.request),
+                                    ISerializeToJsonSummary)()
                     for brain in batch
                 ]
+
+        page = self.context.getDefaultPage()
+        if page:
+            child = self.context._getOb(page)
+            result['default_page'] = getMultiAdapter(
```

Comment:
So we serialize the entire page instead of just one providing a link to the actual default page, right? I first thought about the performance penalty. Though, if no default page is set I guess we should be fine. Could you explain the reasoning behind why you choose to serialize the full object instead of providing a link to the default object?

Reply:
This is to avoid an extra async call.

The idea is this: in Volto View.jsx can select the view component to be used based on this algorithm: 
```
    const RenderedView =
      this.getViewByType() || this.getViewByLayout() || this.getViewDefault();
```
If for some reason it finds that it should use the "default page content", then it would have to do an extra fetch for the content at that location. If that content is already in ``default_page`` attribute, then it can use that to render the proper view implementation.